### ISSUE TITLE
Broken recurring todo form

### ISSUE THREAD

**Issue by maddentim** [TracksApp/tracks#2516]
When I create a new recurring todo, the form is compressed and to the right. I tested on Firefox and Chrome. I have not dug into it enough to provide a good patch as yet. I wanted to get some comfirmation that it was not just my install of tracks. I am running Tracks 2.5.1 and the attached screenshot is from a Macbook pro. I was able to override the width in my browser inspect from auto to 100% which made it usable but not real pretty. The width: auto was an inline style so not a compiled css.

Are you seeing this as well? I can look at creating a patch.

![Screenshot_2020-10-29 TRACKS Recurring Actions](https://user-images.githubusercontent.com/163055/97600471-17081200-19d7-11eb-8e05-b08c317e19b5.png)

**Comment by ZeiP** [comment]
I'm not seeing this in my installation. It might have something to do with your installation, but of course there can also be a bug of some sort.

**Comment by ZeiP** [comment]
Could you perhaps try re-building the assets using script/bootstrap or rake assets:precompile?

**Comment by maddentim** [comment]
Hi, I had tried the `bundle exec rake assets:precompile RAILS_ENV=production` and it didn't fix. I ran it again just now and it is fix! Cool. Maybe apache was caching assets before. 
Now the only issue with this dialog that I am seeing is the close button is too small for the text. I see this dialog close button styling is coming from jQuery UI. Maybe they were expect us to use an 'X' instead of the word 'Close'.
![Screenshot_2020-11-10 TRACKS Recurring Actions](https://user-images.githubusercontent.com/163055/98688846-eb6b1d00-2330-11eb-9093-59e326b45056.png)
